client/client-multi/client-c/listfiles/CMakeGen.cmake

```diff
@@ -15,8 +15,8 @@
 #
 
 set (KAA_SOURCE_FILES  ${KAA_SOURCE_FILES}
-                       src/kaa_event.c
-                       src/gen/kaa_profile_gen.c
-                       src/gen/kaa_logging_gen.c
-                       src/gen/kaa_configuration_gen.c
+                       src/kaa/kaa_event.c
```

Comment:
This file is auto-generated, it is created by routine, decribed in CSdkGenerator.java and CEventSourcesGenerator.java. So paths in these sources should be edited as well.